qa/tasks/mgr/dashboard/helper.py

```diff
@@ -722,3 +723,25 @@ def _validate_json(val, schema, path=[]):
         return _validate_json(val, JLeaf(schema), path)
 
     assert False, str(path)
+
+
+def retry(
+        on_exception: Union[Type[Exception], Tuple[Type[Exception], ...]],
+        tries=3,
+        delay=0,
+        logger: Optional[logging.Logger] = None,
+):
```

Comment:
can we use this same `retry` for the mgr failover scenario? basically retrying in the case its failing to establish a connection to an mgr.

Reply:
Yes, we may use this decorator to cover other situations!